src/modules/interractions.js

```diff
@@ -0,0 +1,50 @@
+/* eslint-disable */
+import { todos } from '../index';
+/* eslint-enable */
+
+export const trueFalse = () => {
+  const checkBox = Array.from(document.querySelectorAll('.checkBox'));
+  checkBox.forEach((complete) => {
+    const ind = checkBox.indexOf(complete);
+    complete.addEventListener('change', () => {
+      if (!complete.checked) {
+        complete.nextElementSibling.style.textDecoration = 'none';
+        complete.nextElementSibling.style.opacity = '1';
+        complete.value = false;
+        todos[ind].completed = false;
+      } else {
+        complete.nextElementSibling.style.textDecoration = 'line-through';
+        complete.nextElementSibling.style.opacity = '0.5';
+        complete.value = true;
+        todos[ind].completed = true;
+      }
+      localStorage.setItem('todos', JSON.stringify(todos));
+    });
+  });
+};
+
+const clearCompleted = document.querySelector('#clear-completed');
+
+const del = () => {
+  Array.from(document.querySelectorAll('.checkBox')).forEach((complete) => {
+    if (complete.checked) {
+      console.log('cndncki');
+      complete.parentElement.remove();
+    }
+  });
+};
+
+export const deleteItems = () => {
+  clearCompleted.addEventListener('click', () => {
+    const filterAll = todos.filter((item) => item.completed !== true);
+    todos.length = 0;
+    filterAll.forEach((item) => {
+      todos.push(item);
+    });
+    todos.forEach((todo, id) => {
+      todo.id = id;
+    });
+    localStorage.setItem('todos', JSON.stringify(todos));
+    del();
+  });
```

Comment:
- [ ] Great job so far but I think you still have here small error I added here 4 tasks I deleted the first and the second then I deleted the third and the fourth theyare deleted but not from local storage so after refrshing the page they came back for some reason I think because you put the functionality for the local stroage inside the function try to sperate it and call it this might solve the problem 💯💪🏻🚀


## I will show a video for explaination

https://user-images.githubusercontent.com/58553711/193430579-ed08c815-27de-455d-9593-2dba19b246a8.mp4